back/tests/Feature/Message/Store/StoreAdministratorTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Tests\Feature\Message\Store;
+
+use App\Models\Role;
+use Illuminate\Http\JsonResponse;
+use Tests\TestCaseWithAuth;
+
+class StoreAdministratorTest extends TestCaseWithAuth
+{
+    protected $username = Role::ADMINISTRATOR;
+
+    /**
+     * @group message
+     */
+    public function testAdministratorCreatingMessage()
+    {
+        $message = [
+            'title'     => 'Test Store Administration',
+            'content'   => "This is a test ",
+            'link'      => 'https://numeris-isep.fr',
+            'is_active' => False,
```

Comment:
Tu as écrit `False` au lieu de `false`
```suggestion
            'is_active' => false,
```